src/index.js

```diff
@@ -34,23 +35,24 @@ app.use(
     })
 );
 
+
 if (!isProduction) {
     app.use(errorhandler());
 }
 
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
-}
+// if (isProduction) {
+//     mongoose.connect(process.env.MONGODB_URI);
+// } else {
+//     mongoose.connect("mongodb://localhost/conduit");
+//     mongoose.set("debug", true);
+// }
 
 require("./models/User");
 
 app.use(require("./routes"));
```

Comment:
Please check this it not suppose to be that way.. i cant suggest anything because i dont know how the structure of things will be as pertaining to the routes.. the `require` keyword is **### es5**